Code/GraphMol/MolDraw2D/MolDraw2DSVG.h

```diff
@@ -86,6 +86,31 @@ class RDKIT_MOLDRAW2D_EXPORT MolDraw2DSVG : public MolDraw2D {
   void initDrawing() override;
   void initTextDrawer(bool noFreetype) override;
 
+  template <class t_obj>
+  void outputTagClasses(const t_obj *obj) const {
+    if (!d_activeClass.empty()) {
+      d_os << " " << d_activeClass;
+    }
+    if (obj->hasProp("_tagClass")) {
+      std::string value;
+      obj->getProp("_tagClass", value);
+      std::replace(value.begin(), value.end(), '\"', '_');
```

Comment:
Why are you replacing `"` with `_` instead of `&quot;`?

Reply:
Because nor "&" neither ";" is a valid css class name or selector character. As far as I know it would be impossible to use such name within browser.

Strictly speaking I should do more detailed sanitation but I got lazy...

Reply:
ah, right, sorry. I forgot that these are going to end up being class names and not attribute values.